units/patch_intent.go

```diff
@@ -501,7 +501,7 @@ func setTasksToPreviousFailed(patchDoc, previousPatch *patch.Patch, project *mod
 	for _, vt := range previousPatch.VariantsTasks {
 		tasksInProjectVariant := project.FindTasksForVariant(vt.Variant)
```

Comment:
Now that we're passing project into getPreviousFailedTasksAndDisplayTasks, we can just move this call into that function rather than pass it in

Reply:
moved